storybook/stories/contributing/contributing-documentation.txt

```diff
@@ -0,0 +1,77 @@
+import { Meta } from '@storybook/addon-docs';
+
+<Meta title="Contributing/Documentation">
+
+This website is composed of articles with guidance on how we create products, which we call documentation.
+
+Our components and patterns need detailed documentation for usage, style, code, and accessibility — that way everyone can easily use them to do great work.
+
+We’d love for you to help us improve our documentation. It’s easy to get started.
+
+*Is this your first contribution? You’ll first need to follow <Link story="Contributing/Get Ready to Contribute">a couple of steps to get started</Link>.*
+
+### **Small copy changes**
+
+If you see a spelling, punctuation, or grammatical error anywhere in the documentation, you can help fix it. 
```

Comment:
These trivial edits definitely don't need an issue opened beforehand — it can go direct to a PR.

Reply:
Opening an issue is easier than creating a PR for people who aren't comfortable with Git — especially in the current workflow that requires forking and cloning the entire repo. I think having this option allows more people to contribute, but I'll rewrite everything to make it clear that if you're comfortable creating a PR, that you can just do that :)

Ideally there'd be an "edit page" button that allows someone to directly create a PR with any copy edits, but I don't think that's possible with the current setup.

Reply:
It should be pretty easy to make copy edits without forking/cloning locally. Edits can go straight to PR in the browser (for example using [this link](https://github.com/WordPress/gutenberg/edit/trunk/storybook/stories/tokens/color.mdx)).